### DISCUSSION TITLE
How to use {{#contains}} with multiple condition checks

### DISCUSSION THREAD

**Discussion by GangadharMFord** [microsoft-search/pnp-modern-search#2517] [Q&A] [answered]
Hi Team,

As part of the PNP search results enhancement side, would like to apply a certain theme few list URL's (like VPN enabled results, OneDrive results) and tried with below code but couldn't worked as expected any idea or help on the same.

Example: {{#contains "['www.google.com','www.hotmail.com']" (slot item @root.slots.PreviewUrl)}}

but the above code or Contains not working with multiple condition checks and please suggest any other approach we have.

We can use Edit result type functionality but there also where I could only test one URL at time. any idea.
thank you,

Regards,
Gangadhar

**Comment by wobba** [comment, marked as answer]
I would look into using a regex with test https://github.com/helpers/handlebars-helpers#test

**Reply by GangadharMFord** [reply]
thank you for your replay and {{Test}} will work again to solve single URL to match but in my case, I want to apply or check incoming url with the certain list of URLs.
hence, I thought the below check will might work but it was failing.
 {{#contains "['www.google.com','www.hotmail.com']" (slot item @root.slots.PreviewUrl)}}

Any other options that will check the previewURL with certain list of uses to match get it.

Regards,
Gangadhar

**Reply by wobba** [reply]
I didn't try it, but something like:

`{{test  (slot item @root.slots.PreviewUrl) (toRegex "www\.(google|hotmail)\.com")}}`

or simpler if you don't need full domain checks

`{{test  (slot item @root.slots.PreviewUrl) (toRegex "google|hotmail")}}`

**Reply by GangadharMFord** [reply]
thank you @wobba for your help and the solution well suited and fixed the concern.
